crawl-ref/source/dat/database/miscname.txt

```diff
@@ -314,4 +314,112 @@ You hear your own voice laughing maniacally!
 
 You hear your own voice screaming in agony!
 
+
```

Comment:
Perhaps I will put these in their own file instead of filling up miscname.txt. "epilogues.txt" maybe.